### PR TITLE
Giskard Server CLI

### PR DESCRIPTION
**Pull request by Googleton**

### CODE REVIEW

File: python-client/giskard/commands/cli_server.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    with open(giskard_settings_path, "w") as f:
+        yaml.dump(settings, f)
+
+
+def _get_settings():
+    # Check the file exists first
+    if not os.path.isfile(giskard_settings_path):
+        return None
+
+    # TODO: Maybe cache it ?
```
Comment (andreybavt) [inline python-client/giskard/commands/cli_server.py:185]: I don't think we should, it's not a heavy operation to read it and there's no cache invalidation problem to handle

**Review by andreybavt — CHANGES_REQUESTED** [review]
I think it's also worth adding a test for _fetch_latest_tag to ensure that we detect a case when the tag format accidentally changes

File: python-client/giskard/commands/cli_server.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+import click
+import requests
+import yaml
+
+import docker
+
+giskard_home_path = os.path.expanduser("~/giskard-home")
+giskard_settings_path = giskard_home_path + "/giskard-settings.yml"
+
+client = docker.from_env()
```
[the remark this answers is not in the document]
Comment (andreybavt) [reply python-client/giskard/commands/cli_server.py:13]: We should also gracefully handle the situation when docker isn't installed/running

**Comment by andreybavt** [comment]
In the current implementation neither `pgdata` nor `giskard-home` aren't persisted in docker volumes. In this case whenever you delete the container (by upgrading Giskard for example), the data will be lost

**Comment by Googleton** [comment]
Pushed volume handling but there is an issue: 

Supervisord uses a small script to init the database if it doesn't exist (checking if the folder at $PGDATA exists). Since I use volumes now, the condition is always false, which means it tries to do pgsql's initdb over an existing folder, fails and the db won't start. Looking for a fix

**Pull request merged**